src/app/shared/form-field/form-field.ts

```diff
@@ -17,7 +18,8 @@ export class TextFormField extends FormField {
        public label: string, 
        public defaultValue: string = '',
        public placeholder: string = '',
-       public validations: Validator[] = []) {
+       public validations: Validator[] = [],
+       public tooltip: Tooltip = new Tooltip()) {
```

Comment:
http://stackoverflow.com/questions/30734509/how-to-pass-optional-parameters-in-typescript-while-omitting-some-other-optional